dynamic.js

```diff
@@ -134,7 +134,7 @@ module.exports = function (fastify, opts, next) {
       const schema = transform
         ? transform(route.schema)
         : route.schema
-      let path = route.url.startsWith(basePath)
+      let path = route.url.startsWith(basePath) && !process.env.FASTIFY_SWAGGER_LEAVE_BASE_PATH
```

Comment:
Could you move this behind a plugin a plugin option?

Reply:
I'm not sure I understand. Are you asking to make this into a plugin or to make this controlled by a passed-in option?

Reply:
I've made some changes that I hope address what you were requesting. Please let me know what you think.